static/bidder-params/mobkoi.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Mobkoi Adapter Params",
+  "description": "A schema which validates params accepted by the Mobkoi adapter",
+  "type": "object",
+  "properties": {
+    "placementId": {
+      "type": "string",
+      "description": "Placement ID"
+    },
+    "adServerBaseUrl": {
+      "type": "string",
+      "description": "Mobkoi's ad server url, optional"
```

Comment:
Nitpick: We don't need `optional` in the description as `required` field clearly depicts only `placementId` is mandatory